tools/dds/dds-server/lrs-device-manager.cpp

```diff
@@ -0,0 +1,119 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+#include <algorithm>
+#include <iostream>
+#include "lrs-device-manager.h"
+
+using namespace tools;
+
+class lrs_device_manager::lrs_sensor_streamer
+{
+public:
+    lrs_sensor_streamer() = default;
+    lrs_sensor_streamer( rs2::sensor rs2_sensor,
+                    rs2::stream_profile stream_profile,
+                    std::function< void( const std::string & stream_name, uint8_t * data, int size ) > cb )
```

Comment:
`data` should be `const`, I think.
The size should preferrably be `size_t`

Reply:
We lose things when we do not pass the frame itself (e.g., metadata). Here we're in the realm of librs, so why do we not just pass the frame?